webapp/src/services/api/studies/raw/index.ts

```diff
@@ -15,28 +15,68 @@
 import client from "../../client";
 import type {
   DeleteFileParams,
-  DownloadMatrixParams,
-  ImportFileParams,
+  GetMatrixFileParams,
+  RawFile,
+  UploadFileParams,
 } from "./types";
 
-export async function downloadMatrix(params: DownloadMatrixParams) {
+/**
+ * Reads a matrix file from a study's raw files.
+ *
+ * @param params - Parameters for reading the matrix
+ * @param params.studyId - Unique identifier of the study
+ * @param params.path - Path to the matrix file
+ * @param params.format - Optional export format for the matrix
```

Comment:
The bracket notation is outdated and provides no value here since we rely on TypeScript interfaces to define optional parameters. Modern TypeScript/JSDoc doesn't use this syntax and you should only refer to Typescript to get this info